### PR TITLE
Search styling

### PR DESCRIPTION
**Pull request by michaltakac**
- [x] Input styling
- [x] Autocomplete styling
- [x] Search icon on the right side of input
- [x] Text highlighting based on search input

screen shot 2017-12-14 at 13 09 02

screen shot 2017-12-14 at 15 13 11

### CODE REVIEW

File: webapp/src/components/Search.css [hunk 1]
```diff
@@ -5,28 +5,67 @@
...
+
+.Search:before {
+  content: "";
+	display: block;
+	background: url("/search-icon.svg") no-repeat;
+	width: 20px;
+	height: 20px;
+	position: absolute;
+	right: 15px;
+	top: 15px;
```
Comment (nicosantangelo) [inline webapp/src/components/Search.css]: This looks like tabs.._gasp_

**Comment by nicosantangelo** [comment]
Looking good so far!

I think the left on the autocomplete menu it's not necessary. It was there in an early version because we had a button next to the search bar:

image

**Comment by michaltakac** [comment]
I followed this mockup - https://projects.invisionapp.com/share/PKETDHTWZ#/screens/267758824 - so changed the width of autocomplete a bit.

...and omg those tabs, vscode is playing with me.

**Comment by michaltakac** [comment]
...and idk why autohighlighting isn't working how it's advertised in docs: https://github.com/reactjs/react-autocomplete#autohighlight-boolean-optional

**Comment by michaltakac** [comment]
Text highlighting based on search input

screen shot 2017-12-14 at 21 18 37

Went with https://github.com/bvaughn/react-highlight-words.

**Pull request merged**